src/network/networkd-route.c

```diff
@@ -124,6 +123,7 @@ int route_new_static(Network *network, const char *filename, unsigned section_li
 
         *ret = route;
         route = NULL;
+        n = NULL;
```

Comment:
No. Let's say that `hashmap_put` fails. Now you have a double free of `n`. The proper fix is to replace `n` by `route->section` in line 116 (and elsewhere if needed).